app.js

```diff
@@ -392,3 +392,36 @@ export default class App {
     renderer.setAnimationLoop(animate);
   }
 }
+
+window.addEventListener("load", () => {
```

Comment:
We cannot do this.

App.js is dynamically imported in the map and I think in that case we do *not* want to `bootstrapFromUrl`. But if we do it this way the bootstrap will happen and mess up the intended usage of `app.js`.